test/unit/utils/StreamMessageValidator.test.ts

```diff
@@ -170,6 +201,49 @@ describe('StreamMessageValidator', () => {
             await getValidator().validate(msg)
         })
 
+        it('accepts unencrypted messages if Brubeck validation is required and the stream public', async () => {
+            getStream = sinon.stub().resolves({
+                ...defaultGetStreamResponse,
+            })
+
+            msg.encryptionType = StreamMessage.ENCRYPTION_TYPES.NONE
+            msg.getPublisherId = () => { return '0x0000000000000000000000000000000000000000' }
```

Comment:
This line is prolly not needed. The identity of the publisher is not validated per se.